includes/payment-methods/class-wc-stripe-upe-payment-method-cc.php

```
@@ -91,16 +91,18 @@ public function get_retrievable_type() {
 	 * @return WC_Stripe_Payment_Token_CC
 	 */
 	public function create_payment_token_for_user( $user_id, $payment_method ) {
+		$pm    = new WC_Stripe_Payment_Method( $payment_method );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '94', column '16'). (undefined)

(MissingImport)